packages/cli/src/commands/optimize/optimize.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @flow
+ */
+
+const fs = require('fs').promises;
+const path = require('path');
+import sharp from 'sharp';
+import type {ConfigT} from 'types';
+import optimizeCommandLineArgs from './optimizeCommandLineArgs';
+import {logger} from '@react-native-community/cli-tools';
+
+function isImage(filePath: string) {
+  const regex = /\.(png|jpg|jpeg)/;
+  return filePath.toLowerCase().match(regex);
+}
+
+async function optimize(args: Array<string>, ctx: ConfigT) {
+  logger.log('Optimizing assets...');
+  const rootPathForImages = path.join(ctx.root, 'src');
+  const images = await getAllImageAssets(rootPathForImages);
+  for (const image of images) {
+    const buffer = await sharp(image).toBuffer();
+    const metadata = await sharp(buffer).metadata();
+    const newName = createOutputFilename(image);
+
+    const {mTimeMS: lastModified, ino: inode} = fs.statSync(image);
+    if (metadata.format === 'jpeg') {
+      sharp(image)
+        .jpeg({quality: 75})
```

Comment:
The quality is configurable through `--quality` flag, please make it so

Reply:
quality is configurable